README.md

```diff
@@ -671,3 +671,91 @@ metric-config.<metricType>.<metricName>.<collectorType>/interval: "30s"
 
 The default is `60s` but can be reduced to let the adapter collect metrics more
 often.
+
+## ScalingSchedule Collectors
+
+The `ScalingSchedule` and `ClusterScalingSchedule` collectors allow
+collecting time-based metrics from the respective CRD objects specified
+in the HPA.
+
+### Supported metrics
+
+| Metric | Description | Type | K8s Versions |
+| ---------- | -------------- | ------- | -- |
+| ObjectName | The metric is calculated and stored for each `ScalingSchedule` and `ClusterScalingSchedule` referenced in the HPAs | `ScalingSchedule` and `ClusterScalingSchedule` | `>=1.12` |
```

Comment:
This should probably be `>=1.16` since the CRD is `apiextensions.k8s.io/v1` which is only supported since `1.16`.

Reply:
I was not sure about this one, thanks a lot for checking.